models/src/main/protobuf/CasperMessage.proto

```diff
@@ -2,9 +2,7 @@
  * The main API is `DeployService`.
  */
 syntax = "proto3";
-package coop.rchain.casper.protocol;
-
-import "google/protobuf/empty.proto";
+package deploy;
```

Comment:
Maybe `deploy` and `propose` namespace should just be `casper`?

Reply:
Yes, I believe that we can do this.